analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -257,8 +257,8 @@ abstract class PackageManager(
                     val errors = listOf(
                         createAndLogIssue(
                             source = managerName,
-                            message = "Resolving dependencies for '${definitionFile.name}' failed with: " +
-                                    e.collectMessagesAsString()
+                            message = "Resolving dependencies for '${definitionFile.relativeTo(analysisRoot)}' " +
```

Comment:
I believe you also need `.invariantSeparatorsPath` here for the test on Windows to succeed.